ioc_rix_sp1k1_calc/ioc_rix_sp1k1_calc.py

```diff
@@ -1,10 +1,39 @@
+import importlib.util
+import sys
+
 import numpy as np
 
 from caproto import ChannelData, ChannelType
 from caproto.asyncio.client import Context
 from caproto.server import AsyncLibraryLayer, PVGroup, pvproperty
 
+# Ignore motor moves smaller than this number
 DEADBAND = 0.05
+# Load path for scientist-modifiable utilities file
+DYNAMIC_PATH = "/cds/home/opr/rixopr/scripts/rix_utilities.py"
+DYNAMIC_NAME = "rix_utilities"
+
+
+class NoneRixDB:
+    """
+    Replacement for rix.db hutch-python imports.
+
+    We need to ignore any such imports if they are present in rix_utilities.py
+    because this SIOC needs to start up without loading the entire rix
+    beamline.
+    """
+    def __getattr__(self, name):
+        return None
+
+
+# Fool Python into thinking rix.db has already been imported and is this
+sys.modules["rix.db"] = NoneRixDB()
```

Comment:
For the unfamiliar, this is here because `rix_utilities.py` holds functions that need the `rix.db` hutch-python import.  I think it'd be nice if we could separate the IOC calculation functions from the rest of these utilities, but that's not really a problem for this PR.  

I forsee some other import being added to `rix_utilities.py` that breaks this IOC in the future.

Reply:
I agree, this is a concern. If it breaks again I'll need a more clever workaround... Or we can ask them to change how their module works and separate out the calcs into a very pure calculation-only module.

fwiw the file doesn't even *need* to import rix.db stuff, it could encapsulate these inside function calls for default arguments for example...